panel/pane.py

```diff
@@ -476,6 +476,38 @@ def _imgshape(cls, data):
         return int(w), int(h)
 
 
+class SVG(Image):
+
+    imgtype = 'svg'
+
+    @classmethod
+    def applies(cls, obj):
+        return (super(SVG, cls).applies(obj) or
+                (isinstance(obj, basestring) and obj.lstrip().startswith('<svg')))
```

Comment:
What does `(super(SVG, cls))` do here?

Reply:
Adds support for all the usual Image types, e.g. local SVG files, remote SVG files and objects with a `_repr_svg_`.

Reply:
Oh, I see now by looking at the superclass, which I think I wrote; it uses the `imgtype` to allow it to do general checking but specialized for this specific type. :-)  Makes sense now.